### ISSUE TITLE
Include Firefox driver 

### ISSUE THREAD

**Issue by wout** [luckyframework/lucky_flow#161]
In true Lucky spirit (aka  "it just works"), having the Firefox driver included in the repo would be nice. Or at least documented. Using it would then be just a matter of changing a keyword for `LuckyFlow.default_driver`.

Here's what I came up with. Not sure if it's 100%, but it works as expected:

```crystal
module LuckyFlow::Selenium
  class Firefox::Driver < Driver
    private getter driver : ::Selenium::Driver do
      service = ::Selenium::Service.firefox(driver_path: driver_path)
      ::Selenium::Driver.for(:firefox, service: service)
    end

    def initialize(&)
      super ::Selenium::Firefox::Capabilities.new
      yield @capabilities.as(::Selenium::Firefox::Capabilities)
    end

    private def driver_path
      LuckyFlow.settings.driver_path || Webdrivers::Geckodriver.install
    rescue err
      raise DriverInstallationError.new(err)
    end
  end
end

LuckyFlow::Registry.register :firefox do
  LuckyFlow::Selenium::Firefox::Driver.new { }
end

LuckyFlow::Registry.register :headless_firefox do
  LuckyFlow::Selenium::Firefox::Driver.new do |config|
    config.firefox_options.args = ["--headless"]
  end
end
```

Let me know if you'd like a PR.

Sidenote: I was forced to use Firefox because Chrome kept giving me `element click intercepted: Element is not clickable at point (x, y)`. I tried a bunch of things without success, including different versions of Chrome.

**Comment by jwoertink** [comment]
I'm down to add it in if you wanted to submit a PR. That would be awesome!

**Comment by jwoertink** [comment]
Fixed by #162 

Thanks!
